src/templates/home/home.css

```diff
@@ -0,0 +1,359 @@
+/* Style for Home Page */
+
+.homeTitle {
+    font-size: 23px;
+    color: var(--main-color);
+}
+
+
+/* Title Image */
+
+.bodyTitleImage {
+    background-image: url('https://www.dissco.eu/wp-content/uploads/woods-bg.jpg');
+    height: 400px;
+    margin-top: -100px;
+    position: absolute !important;
+    z-index: -1;
+}
+
+
+/* Search Bar */
+
+.homeSearchBar {
+    z-index: 1;
+    position: relative;
+    margin-top: 280px !important;
+}
+
+.searchBarGreetingBlock {
+    height: 125px;
+    background-color: var(--main-color);
+    color: white;
+    border-top-left-radius: 8px;
+    border-bottom-left-radius: 8px;
+    -webkit-box-shadow: 5px 2px 11px 2px rgba(0, 0, 0, 0.54);
+    box-shadow: 5px 2px 11px 2px rgba(0, 0, 0, 0.54);
+}
+
+.searchBarGreeting {
+    font-size: 18px;
+    text-align: center;
+}
+
+.searchBarBlock {
+    height: 125px;
```

Comment:
I think certain components like this one can have a static height, height in percentages is very gruesome to work with, specially on the homepage which can extend further to the bottom than the static height pages like search and annotate. On these pages it is way easier to use percentages for height because of the static height. Tested the pixel height of the search bar on other mobile devices and looks to be fine (aside from the width which needs to be set for smaller screens in general)